go.mod

```diff
@@ -1,20 +1,23 @@
 module github.com/openshift/configure-alertmanager-operator
```

Comment:
Why are we bumping the golang version and all these dependencies in this PR?

Reply:
This PR is adding the initial refactored suite of tests from osde2e. This is a requirement of using the framework we are writing.

Why are we staying on an end of life version of Go?